tests/baseline/tsfresh_feature_names.py

```diff
@@ -0,0 +1,229 @@
+TSFRESH_BASELINE_VERSION = '0.3.0'
```

Comment:
Maybe I have to understand the reason: why did you find it useful to have two copy around? If there is a good reason, I am fine with that! :-)